spyderlib/config.py

```diff
@@ -477,6 +477,7 @@ def is_ubuntu():
               '_/save current layout': "Shift+Alt+S",
               '_/toggle default layout': "Shift+Alt+Home",
               '_/layout preferences': "Shift+Alt+P",
+              '_/restart': "Ctrl+R",

```

Comment:
A more "complicated" shortcut is needed, Ctrl+R is too likely to be triggered by accident. In some editors this is the shorcut fo "Redo" for example. Maybe Ctrl+Shift+R ?